cibuildwheel/resources/build-platforms.toml

```diff
@@ -115,3 +115,8 @@ python_configurations = [
   { identifier = "pp38-win_amd64", version = "3.8", arch = "64", url = "https://downloads.python.org/pypy/pypy3.8-v7.3.11-win64.zip" },
   { identifier = "pp39-win_amd64", version = "3.9", arch = "64", url = "https://downloads.python.org/pypy/pypy3.9-v7.3.11-win64.zip" },
 ]
+
+[pyodide]
+python_configurations = [
+  { identifier = "cp311-emscripten_3_1_32", version = "3.11.2", pyodide_version = "0.23.2", emscripten_version = "3.1.32" },
```

Comment:
So, if we think the output wheels will be called pyodide, shall we change the build identifier to `cp311-pyodide_wasm32`? I like to keep the build identifiers stable. Or do we need to wait for that decision do you think?

Reply:
Yeah we could do that.